src/components/projects/view/tag-selector.svelte

```diff
@@ -1,21 +1,20 @@
 <script>
   import { createEventDispatcher } from 'svelte';
-  import Button from 'ui/button.svelte';
-  import Autocomplete from 'ui/autocomplete.svelte';
+  import { Button, Autocomplete, Chip, Dot } from 'attractions';
   import Labeled from 'ui/labeled.svelte';
-  import UnclickableChip from 'ui/unclickable-chip.svelte';
 
   export let tags;
-  export let value;
+  export let value = [];
   let valueTags = value.map(id => tags.find(tag => tag.id === id));
   let editing = false;
 
   // eslint-disable-next-line require-yield
   async function *getOptions(query) {
+    let output = tags;
     if (query) {
-      return tags.filter(tag => tag.name.toLowerCase().includes(query.toLowerCase()));
+      output = output.filter(tag => tag.name.toLowerCase().includes(query.toLowerCase()));
     }
-    return tags;
+    return output.filter(tag => !value.includes(tag.id));
```

Comment:
Didn't you mention in some other PR that generator functions have to always `yield` instead of `return`?

Reply:
Good call